bin.js

```diff
@@ -0,0 +1,29 @@
+#! /usr/bin/env node
+
+'use strict'
+
+var pretty = require('./pretty')
+
+module.exports = pretty
+
+if (require.main === module) {
```

Comment:
we no longer need this check - safe to remove since we won't be requiring bin.js